components/src/theme/theme.ts

```diff
@@ -47,6 +47,25 @@ export const theme = {
       contrast: '#FFFFFF',
     },
   },
+  guidelines: {
+    list: {
+      fontFamily: 'sans serif',
+      lineHeight: 'action',
+      width: 857,
+      marginLeft: '0.5rem',
+      fontSizes: 4,
```

Comment:
Some observations:
- The width value shouldn't be set, because it's relative to where the component is placed. So you can remove it from here :rocket:
- For the marginLeft, you can use one of the values defined in the spaces. Keep in mind that `1rem == 16` :straight_ruler:
- Notice how there should be a bigger space between the last divider and the section (check the screenshot below). You can add a paddingTop here to add the 8px difference
![image](https://user-images.githubusercontent.com/20714148/97115745-2e918300-16d7-11eb-8714-90efc1ba35a0.png)
- The attribute here should be fontSize instead of fontSizes, and the value should be 2, because the font size for the items is 16px
- Remember to define the font color here (I think it's muted.0)

Reply:
Okay, solved! :rocket: